platform/iphone/app_delegate.mm

```diff
@@ -671,6 +671,7 @@ - (void)applicationDidFinishLaunching:(UIApplication *)application {
 												  isAdvertisingTrackingEnabled]];
 
 #endif
+	return TRUE;
```

Comment:
There are some early returns above where you should return FALSE.

Reply:
I commented it on https://github.com/godotengine/godot/pull/14355#discussion_r155456196
there is no change to hit there because there's no "game_analytics" module anymore, don't know since when.